aiida_mlip/parsers/sp_parser.py

```diff
@@ -98,7 +98,9 @@ def parse(self, **kwargs) -> int:
         with self.retrieved.open(xyz_output, "rb") as handle:
             self.out("xyz_output", SinglefileData(file=handle, filename=xyz_output))
 
-        content = read(Path(self.node.get_remote_workdir(), xyz_output))
+        content = read(
+            Path(self.node.get_remote_workdir(), xyz_output), format="extxyz"
```

Comment:
Why is this necessary?

In general janus allows any file format to be written out. Doesn't this limit that?

Reply:
yeah I thought it would be only xyz files. 
but even if I didn't do this, I'm still reading it as a dictionary, if the output was another format I couldn't do that (? depends on the format I guess)

Reply:
I'm not actually sure how many alternative file formats allow you to write out data from `info`/`arrays`/`calc.results`, but in general if it's stored, it should be readable again.

You can certainly get an `info` dictionary after reading a .cif file, although I'm not sure about how much can be written in the first place.

If it does only make sense to support extxyz, this should documented and probably (`--out`) validated.